### PR TITLE
Add content to readme file

### PR DESCRIPTION
**Pull request by ismailakinkunmi**
- Update readme file content.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,88 @@
...
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to microverse for their support
+- Inspiration
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (zieeco) [inline README.md]: - He great job with the readme, however, there's an issue with the license link as it's broken, please, kindly update the MIT link to make your readme more professional.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,88 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to microverse for their support
+- Inspiration
+
```
Comment (zieeco) [inline README.md]: - Hey, as its a requirement for this project to be approved, please, kindly give credit to [murak](https://www.behance.net/muratk), the author of the design as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) to make your readme more professional.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @ismailakinkunmi,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alfredbis29 — APPROVED** [review]
### Hi @ismailakinkunmi 🙋‍♀️,  

Good JOB! 💯 
You have done a great job setting up your kanban task. 
Congratulations!!!

# Status: Approved ✔️ 

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alfredbis29   in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**